### ISSUE TITLE
GetOrderBookAsync method's parameter is "depth", not "limit"

### ISSUE THREAD

**Issue by pwnzya** [JKorf/Bittrex.Net#176]
Just tested GetOrderBookAsync method, and got an error:
> ResponseStatusCode = BadRequest
> Error = 3
> Server error: BAD_REQUEST
> Details: Refer to the data field for specific field validation failures.
> Data: {"invalidRequestParameter": "limit"}

According to the [doc](https://bittrex.github.io/api/v3#operation--markets--marketSymbol--orderbook-get), the correct parameter name is "depth", not "limit".
> depth: integer (int32) in query
> maximum depth of order book to return (optional, allowed values are [1, 25, 500], default is 25)

In the method GetOrderBookAsync, I think the line
```c#
parameters.AddOptionalParameter("limit", limit?.ToString(CultureInfo.InvariantCulture));
```
should be
```c#
parameters.AddOptionalParameter("depth", limit?.ToString(CultureInfo.InvariantCulture));
```

**Comment by JKorf** [comment]
Oh that's my bad. That's what you get for quickly pushing something without testing it.

Will be fixed, probably tomorrow.

**Comment by JKorf** [comment]
Fixed in 4.1.6.
